satpy/etc/readers/seviri_l2_grib.yaml

```diff
@@ -73,244 +73,274 @@ datasets:
 
   cloud_mask:
```

Comment:
Please add `flag_values` and `flag_meanings`.

Reply:
done